Assets/MixedRealityToolkit/Utilities/Editor/PackageManifest/PackageManifestUpdater.cs

```diff
@@ -23,12 +21,13 @@ internal static void EnsureMSBuildForUnity()
         {
             // Locate the full path to the package manifest.
             DirectoryInfo projectRoot = new DirectoryInfo(Application.dataPath).Parent;
-            string manifestPath = Path.Combine(projectRoot.FullName, PackageManifestRelativePath);
-            
+            string[] paths = { projectRoot.FullName, "Packages", "manifest.json" };
+            string manifestPath = Path.Combine(paths);
```

Comment:
Path.Combine takes a param, you don't have to do this in two step, just pass in all three parameters into it.